dev-util/its4/its4-1.1.1-r2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="ITS4: Software Security Tool"
+HOMEPAGE="http://www.cigital.com/its4/"
+SRC_URI="https://dev.gentoo.org/~robbat2/distfiles/${P}.tgz"
+
+LICENSE="ITS4"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86"
+
+S="${WORKDIR}/${PN}"
+
+src_prepare() {
+	sed -i \
+		-e 's,iostream.h,iostream,g'\
+		"${S}"/configure || die
+	sed -i \
+		-e 's/$(CC) -o/$(CC) $(OPTIMIZATION) $(EXTRA_FLAGS) -o/' \
+		"${S}"/Makefile.in || die
+
+	# Bug 738936 fails to compile with clang/LLVM toolchain
+	sed -i \
```

Comment:
Would you mind using a patch for this instead? It feels a bit fragile.

Reply:
(Feel free to fold the other 2 seds into their own single patch too - i.e. it's okay to combine those two into one, but I'd do config.C in its own.)

Reply:
Sure thing.

Reply:
Done. I downloaded the sources, made a temporary git repo out of it, and applied the sed lines turning them into patches in the process. The patches are created with git format-patch.